### PR TITLE
{vis}[GCCcore/8.3.0] XCrySDen v1.6.2, Togl v2.0, bwidget v1.9.14

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

requires ~~https://github.com/easybuilders/easybuild-easyblocks/pull/1959~~

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/8e866375b380f48ae3650c2fae848343 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/7c0b788480d366090823751532bd2989 for a full test report.

File: easybuild/easyconfigs/b/bwidget/bwidget-1.9.14-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'Tarball'
+name = 'bwidget'
```
Comment (boegel) [inline easybuild/easyconfigs/b/bwidget/bwidget-1.9.14-GCCcore-8.3.0.eb]: style nitpicking: please add an empty line above

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/cf0f0d68e4a4c8514be6f49fb11de21a for a full test report.

File: easybuild/easyconfigs/t/Togl/Togl-2.0_configure.patch [hunk 2]
```diff
@@ -0,0 +1,42 @@
+* remove check for tclInt.h and tkInt.h in source folder, not necessary
+author: Miguel Dias Costa (National University of Singapore0
```
Comment (boegel) [inline easybuild/easyconfigs/t/Togl/Togl-2.0_configure.patch]: typo: `0` -> `)`

File: easybuild/easyconfigs/x/XCrySDen/XCrySDen-1.6.2_no-bwidget.patch [hunk 3]
```diff
@@ -0,0 +1,11 @@
+--- Makefile.orig	2020-01-16 18:58:30.499898000 +0800
```
Comment (boegel) [inline easybuild/easyconfigs/x/XCrySDen/XCrySDen-1.6.2_no-bwidget.patch]: please add missing description + authorship

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/277d8d8677f377b32c1daa231d753b59 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node3104.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/bb354105a8794b818f86ec7dedf10505 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/893357a98f294f549bc43b18b68776f1 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**